.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "docs/_themes"]
 	path = docs/_themes
-	url = git://github.com/mitsuhiko/flask-sphinx-themes.git
+	url = https://github.com/mitsuhiko/flask-sphinx-themes.git
```

Comment:
This is out of scope of this PR, feel free to open a different PR with this if you want it, although you'd need to make a convincing argument for it, because the `git` url seems fine to me.

Reply:
You are correct - we can certainly put this into a separate PR. For background, it's related to a security change on GitHub that took effect earlier this month ( https://github.blog/2021-09-01-improving-git-protocol-security-github/ ).